src/main/java/Duke.java

```diff
@@ -1,10 +1,64 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
+        String line = "____________________________________________________\n";
         System.out.println("Hello from\n" + logo);
+        System.out.print(line + "Hello! I'm Duke\n" + "What can I do for you?\n" + line);
+        Task[] task = new Task[100];
```

Comment:
since the number of task can be more than 100. you can consider to use the dynamic size array list